selfdrive/car/gm/interface.py

```diff
@@ -112,6 +114,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
     ret.radarTimeStep = 0.0667  # GM radar runs at 15Hz instead of standard 20Hz
 
     if candidate == CAR.VOLT:
+      ret.minEnableSpeed = -1
```

Comment:
Careful that some older Volts fault if you engage under a certain speed for the first time, I'd leave this. Or at the least, detect this somehow

Reply:
Thanks for the catch -- didn't mean to include that change in the commit.